src/createAsyncThunk.ts

```diff
@@ -220,6 +248,16 @@ If you want to use the AbortController to react to \`abort\` events, please cons
       const promise = (async function() {
         let finalAction: ReturnType<typeof fulfilled | typeof rejected>
         try {
+          if (
+            options &&
+            options.condition &&
+            !options.condition(arg, { getState, extra })
```

Comment:
I think we should make this a strict `=== false` comparison.  The user might return undefined or falsy by accident.  The assumption should be that the thunk is going to run unless they really say no.

Reply:
Good point.